pubspec.yaml

```diff
@@ -54,6 +54,11 @@ dependencies:
   package_info_plus: ^8.0.0
   ## Multi Formatter
   flutter_multi_formatter: ^2.12.0
+  flutter_timezone: ^2.1.0
+  device_calendar:
```

Comment:
add a comment above this line about why we're using a git source, e.g. is there a particular pull request we're waiting to get merged?

Reply:
@IguJl15 please address this in a follow-up PR